abcclassroom/gitutils.py

```diff
@@ -12,7 +12,29 @@
 
 import github3 as gh3
```

Comment:
i thought about this more @kcranston i'd like this still to be called github not gitutils. can you please rename it.